### PR TITLE
Tech: gogdl update 

### PR DESCRIPTION
**Pull request by imLinguin**
Updated gogdl and added a check if gog redist directory exists - in order to make sure ISI is always installed

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by Etaash-mathamsetty — APPROVED** [review]
lgtm

File: src/backend/downloadmanager/downloadqueue.ts [hunk 1]
```diff
@@ -142,7 +144,7 @@ async function addToQueue(element: DMQueueElement) {
       'dependencies' in installInfo.manifest
     ) {
       const newDependencies = installInfo.manifest.dependencies
-      if (newDependencies?.length) {
+      if (newDependencies?.length || !existsSync(gogRedistPath)) {
```
Comment (Etaash-mathamsetty) [inline src/backend/downloadmanager/downloadqueue.ts:147]: Looking over this again shouldn't this be an and?

Comment (imLinguin) [reply src/backend/downloadmanager/downloadqueue.ts:147]: we are about to create that path, it's a check to make sure it's not removed and we always have ISI downloaded

Comment (imLinguin) [reply src/backend/downloadmanager/downloadqueue.ts:147]: I might need to add a fallback to newDependencies in case it's undefined

**Review by arielj — APPROVED** [review]

**Pull request merged**